tests/integration/tests/querying.test.ts

```diff
@@ -0,0 +1,403 @@
+import { describe, expect, test } from "bun:test";
+import {
+	Duration,
+	GeometryCollection,
+	GeometryLine,
+	GeometryMultiPolygon,
+	GeometryPoint,
+	GeometryPolygon,
+	RecordId,
+	StringRecordId,
+	Table,
+	Uuid,
+} from "../../../src";
+import { setupServer } from "../surreal.ts";
+
+const { createSurreal } = await setupServer();
+
+type Person = {
+	id: RecordId<"person">;
+	firstname: string;
+	lastname: string;
+	age?: number;
+};
+
+describe("create", async () => {
+	const surreal = await createSurreal();
+
+	test("single", async () => {
+		const single = await surreal.create<Person, Omit<Person, "id">>(
+			new RecordId("person", 1),
+			{
+				firstname: "John",
+				lastname: "Doe",
+			},
+		);
+
+		expect(single).toMatchObject({
```

Comment:
I think to match object ignore additional keys which I don't think is what you should use here